python/lsst/meas/base/plugins.py

```diff
@@ -695,6 +701,16 @@ def getExecutionOrder(cls):
 
     def __init__(self, config, name, schema, metadata):
         SingleFramePlugin.__init__(self, config, name, schema, metadata)
+
+        # Check that the required columns are already in the schema.
+        if not all(("slot_Shape" in schema.getAliasMap().keys(),
+                    "slot_PsfShape" in schema.getAliasMap().keys(),
```

Comment:
Do you have to get the alias map here, or can you just ask `"slot_Shape" in schema`? I just checked and that appears to do what you'd want.

Reply:
You'd have had to do `"slot_Shape_xx" in schema` and not what you typed, right? I could go either way, but I inclined towards the current implementation since it felt more generalizable without putting in the extra `_xx` by hand.